### ISSUE TITLE
Using mustache for live demo (unescape JSON data)

### ISSUE THREAD

**Issue by badlydrawnrob** [badlydrawnrob/anki#27]
**TL;DR**

- **Stick to plain HTML and Markdown**
    + **[Be careful what code you're adding to Anki fields](http://shebang.brandonmintern.com/foolproof-html-escaping-in-javascript/)!**
    + Avoid javascript in fields as it may have unintended consequences
- **[Make sure you escape your HTML](https://stackoverflow.com/a/22551692)** if you're making your own (local) card preview
    + You can use an [online tool](https://www.freeformatter.com/json-escape.html)

**More info**

Anki uses [Mustache](https://mustache.github.io/mustache.5.html), or [Mustache-like template tags](https://apps.ankiweb.net/docs/manual.html#basic-templates). The [javascript library](https://github.com/janl/mustache.js/) is the closest representation of the Anki themes I can find.

- `{{{☆ Syntax}}}` **Triple brackets are used**, to unescape json html **in the live demo cards**
    + Without it, syntax blocks are interpreted as strings (not raw html)
    + Not ideal, but doesn't seem to have negative consequences in Anki.
- `{{{ ... }}}` **Triple brackets are also used for the compiled Anki theme decks**
- **HTML in json must also be escaped** or they won't compile
    + e.g: `<pre>[0, 3, 6, 9, 12, 15]<\/pre>`
- **Javascript may compile in Anki fields:** `<script>..</script>`
    + Stick to plain Markdown and HTML
    + Be careful if you're importing someone else's (Anki theme) decks
